src/parsers/yaml/yaml-ast-parser.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import defaultParserInterface from '../utils/defaultParserInterface';
+import pkg from 'yaml-ast-parser/package.json';
+import SettingsRenderer from '../utils/SettingsRenderer';
+
+const ID = 'yaml-ast-parser';
+
+export default {
+  ...defaultParserInterface,
+
+  id: ID,
+  displayName: ID,
+  version: pkg.version,
+  homepage: pkg.homepage || 'https://www.npmjs.com/package/yaml-ast-parser',
+
+  _ignoredProperties: new Set(['parent', 'errors']),
+  locationProps: new Set(['startPosition', 'endPosition']),
+
+  nodeToRange(node) {
+    return [node.startPosition, node.endPosition];
+  },
+
+  getNodeName(node) {
+    return node.type;
```

Comment:
hmmm, `node.kind` is a number. I should find the enum mapping for those numbers.